packages/back-end/src/services/features.ts

```diff
@@ -104,8 +104,9 @@ function generateVisualExperimentsPayload({
   const isValidSDKExperiment = (
     e: AutoExperimentWithProject | null
   ): e is AutoExperimentWithProject => !!e;
-  const sdkExperiments: Array<AutoExperimentWithProject | null> = visualExperiments.map(
-    ({ experiment: e, visualChangeset: v }) => {
+  const sdkExperiments: Array<AutoExperimentWithProject | null> = visualExperiments
+    .sort((_a, b) => (b.visualChangeset.urlRedirects.length ? 1 : -1))
```

Comment:
Filter and concat (split into url redirect experiments and visual editor exps and place url redirects first)